recipes-core/psplash/psplash_git.bbappend

```diff
@@ -1,4 +1,5 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
this should be BPN otherwise it may not work in multilib env.

Reply:
++